vllm/model_executor/models/qwen.py

```diff
@@ -0,0 +1,316 @@
+# coding=utf-8
+# Adapted from
+# https://huggingface.co/Qwen/Qwen-7B/blob/main/modeling_qwen.py
+# Copyright (c) Alibaba Cloud.
+# LICENSE: https://huggingface.co/Qwen/Qwen-7B/blob/main/LICENSE
```

Comment:
I noticed that here we introduced a new license different from this project. Will this license bring some potential legal risks for commercial use?

Reply:
Qwen's license allows commercial use. The following is the original text from the Qwen license.

> If you are commercially using the Materials, and your product or service has more than 100 million monthly active  users, You shall request a license from Us. 

I noticed that the baichuan model does not include the license of the original project. Do we need to add Qwen's license in this PR?
> [baichuan's license](https://huggingface.co/baichuan-inc/Baichuan-13B-Base/blob/main/Community%20License%20for%20Baichuan-13B%20Model.pdf):  For commercial use, you need to send an email to apply for a license.

Reply:
Maybe we can ask owners for some advice🤔. 
Hi @WoosukKwon and @zhuohan123, should we follow the model's license when adding new models support in vLLM?

Reply:
I believe we need to follow the model code's license instead of the model's licenses. For [Baichuan](https://github.com/baichuan-inc/Baichuan-13B#%E5%8D%8F%E8%AE%AE) (and LLaMA), the code is Apache 2 but the model has a special license. In this case we can include the code with no problem. However, for Qwen, both its code and model are with their restricted license. I believe we should include the link to Qwen's license on the top of Qwen's source file for safety.